src/test/java/pl/wrapper/parking/facade/domain/ParkingServiceTests.java

```diff
@@ -0,0 +1,96 @@
+package pl.wrapper.parking.facade.domain;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.time.LocalTime;
+
+import java.util.List;
+
+
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import pl.wrapper.parking.facade.ParkingService;
+import pl.wrapper.parking.pwrResponseHandler.PwrApiServerCaller;
+import pl.wrapper.parking.pwrResponseHandler.dto.ParkingResponse;
+
+@WebMvcTest(ParkingController.class)
+@ComponentScan(basePackageClasses = ParkingService.class)
+public class ParkingServiceTests {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @MockBean
+    private PwrApiServerCaller pwrApiServerCaller;
+
+    private Integer correctId;
+
+
+    private static final ObjectMapper om = new ObjectMapper()
```

Comment:
There is a better way do it, let's use the Spring JSONObject and JSONArray instead of ObjectMapper. It will simulate how an external client would work, accessing the properties by json map instead of casting it to our models